src/app/components/audio-analysis/pages/list/list.component.html

```diff
@@ -0,0 +1,55 @@
+<ng-container *ngIf="rows && !error">
+  <h1>Analysis Jobs</h1>
+
+  <p>
+    Displaying <strong>{{ pageSize }}</strong> of
+    <strong>all {{ totalModels }}</strong> analysis jobs.
+  </p>
+
+  <ngx-datatable
+    #table
+    bawDatatableDefaults
+    [columns]="columns"
+    [rows]="rows"
+    [count]="totalModels"
+    [offset]="pageNumber"
+    (page)="setPage($event)"
+    (sort)="onSort($event)"
+  >
+    <ngx-datatable-column name="Name"></ngx-datatable-column>
+    <ngx-datatable-column name="Scripts" [width]="50" [sortable]="false">
+      <ng-template let-value="value" ngx-datatable-cell-template>
+        <baw-inline-list [items]="value.scripts" itemKey="name"></baw-inline-list>
+      </ng-template>
+    </ngx-datatable-column>
+    <ngx-datatable-column name="Creator" [sortable]="false">
+      <ng-template let-value="value" ngx-datatable-cell-template>
+        <baw-user-link [user]="value.creator"></baw-user-link>
+      </ng-template>
+    </ngx-datatable-column>
+    <ngx-datatable-column name="Started" [width]="110" [maxWidth]="110">
+      <ng-template let-value="value" ngx-datatable-cell-template>
+        @if (value) {
+        <baw-datetime [value]="value"></baw-datetime>
+        } @else {
+        <span>Unknown</span>
+        }
```

Comment:
i would have expected baw-datetime should be able to handle null/undefined

Reply:
At the moment it doesn't. It will display "Invalid Date Time"

However, I agree that `null` and `undefined` should display `Unknown`

Reply:
Issue reference: https://github.com/QutEcoacoustics/workbench-client/issues/2212